src/screens/Address/NewAddressScreen.tsx

```diff
@@ -87,7 +87,7 @@ const NewAddressScreen = ({ navigation, ...props }: NewAddressScreenProps) => {
   }
 
   return (
-    <Screen {...props}>
+    <ScrollScreen fill hasHeader verticalGap {...props}>
```

Comment:
Following the discussion in https://github.com/alephium/mobile-wallet/pull/194#discussion_r1321655007, I made these screens scrollable